packages/raystack/components/toast/toast.module.css

```
@@ -66,7 +66,7 @@
   cursor: default;
   z-index: calc(1000 - var(--toast-index));
 
-  transition: opacity 500ms;
+  transition: opacity var(--rs-duration-slow) var(--rs-ease-out);
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)